### ISSUE TITLE
Upgrade notification happens every time, and sometimes says upgraded from v0.0.0

### ISSUE THREAD

**Issue by AshSimmonds** [dendronhq/dendron#2842]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [x] Workspace.  VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup.  Dendron's Lookup Command
- [ ] Views.  Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema.  Dendron Schemas
- [ ] Pod.  Data import from / export to Dendron
- [ ] Publish.  External Site Publish
- [ ] Markdown.  Markdown features, such as syntax support and features

## Describe the bug

As titled.

## To Reproduce

Unknown - apart from just opening VSCode.

Note: I am using 2-3 different machines with different settings a day, and it's something that's just started after I stopped using Dendron for about a month and came back as of v0.91.0 - and I also use both normal and have just started using Insider's Build.

## Expected behavior

Only display the notification toast when it actually upgrades.

## Screenshots

n/a

## Desktop (please complete the following information)

- Windows 10/11
- Dendron 0.91.0 --> 0.92.0
  - VS Code: 1.66.2
  - Insider's Build: 1.67.0-insider

## Dendron Log file

```json
{"level":30,"time":1651098790176,"pid":13924,"hostname":"Flippy-Turd","name":"dendron","ctx":"Logger:configure","msg":"exit","logLevel":"info"}
{"level":30,"time":1651098790266,"pid":13924,"hostname":"Flippy-Turd","name":"dendron","ctx":"_activate","stage":"prod","isDebug":false,"logLevel":"info","logPath":"c:\\Users\\ashsi\\AppData\\Roaming\\Code - Insiders\\logs\\20220428T080307\\exthost1\\dendron.dendron","extensionPath":"c:\\Users\\ashsi\\.vscode-insiders\\extensions\\dendron.dendron-0.92.0","extensionUri":"c:\\Users\\ashsi\\.vscode-insiders\\extensions\\dendron.dendron-0.92.0"}
{"level":30,"time":1651098790292,"pid":13924,"hostname":"Flippy-Turd","name":"SegmentClient","msg":"No cache path for Segment specified. Failed event uploads will not be retried."}
{"level":30,"time":1651098790293,"pid":13924,"hostname":"Flippy-Turd","name":"SegmentClient","msg":"user telemetry setting: enabled by config"}
{"level":30,"time":1651098790293,"pid":13924,"hostname":"Flippy-Turd","name":"SegmentClient","msg":"telemetry initializing"}
{"level":30,"time":1651098790293,"pid":13924,"hostname":"Flippy-Turd","name":"SegmentClient","msg":"using existing id"}
{"level":30,"time":1651098790293,"pid":13924,"hostname":"Flippy-Turd","name":"SegmentClient","msg":"anonymous id","anonymousId":"5qn3p8m27ttx68u5326a9tc"}
{"level":30,"time":1651098790301,"pid":13924,"hostname":"Flippy-Turd","name":"dendron","ctx":"DendronExtension","msg":"initialized"}
{"level":30,"time":1651098790349,"pid":13924,"hostname":"Flippy-Turd","name":"dendron","ctx":"_activate","msg":"initializeWorkspace","wsType":"NONE","currentVersion":"0.92.0","previousGlobalVersion":"0.91.0","extensionInstallStatus":"UPGRADED"}
{"level":30,"time":1651098790350,"pid":13924,"hostname":"Flippy-Turd","name":"dendron","ctx":"_activate","msg":"dendron not active"}
{"level":30,"time":1651098790350,"pid":13924,"hostname":"Flippy-Turd","name":"dendron","ctx":"showWelcomeOrWhatsNew","version":"0.92.0","previousExtensionVersion":"0.0.0"}
{"level":30,"time":1651098790350,"pid":13924,"hostname":"Flippy-Turd","name":"dendron","ctx":"showWelcomeOrWhatsNew","msg":"extension, new version","version":"0.92.0","previousExtensionVersion":"0.0.0"}
{"level":30,"time":1651099417295,"pid":13924,"hostname":"Flippy-Turd","name":"dendron","ctx":"dendron.dev.openLogs:run","msg":"pre-execute"}
{"level":30,"time":1651099417295,"pid":13924,"hostname":"Flippy-Turd","name":"dendron","ctx":"execute","opts":{}}
{"level":30,"time":1651099417305,"pid":13924,"hostname":"Flippy-Turd","name":"dendron","ctx":"dendron.dev.openLogs:run","msg":"post-execute"}
```

**Comment by SeriousBug** [comment]
We actually just released a quick fix to solve this issue, Dendron 0.92.1. Could you let us know if this keeps happening after upgrading?

**Comment by AshSimmonds** [comment]
A few restarts and the update took. Thanks.
